kafka-ui-react-app/src/components/common/SmartTable/SmartTable.tsx

```diff
@@ -48,10 +48,9 @@ export const SmartTable = <T, TId extends IdType, OT = never>({
 
       const { headerCell, title, orderValue } = child.props;
 
-      const HeaderCell = headerCell as
-        | React.FC<TableHeaderCellProps<T, TId, OT>>
-        | undefined;
-
+      const HeaderCell = headerCell as React.FC<
+        TableHeaderCellProps<T, TId, OT>
+      >;
```

Comment:
one Question why did we put the undefined in the older version

Reply:
@Mgrdich I'm not sure why but that caused problem for sonar and there was suggestion from sonar to remove that part as it's not changing the type
<img width="676" alt="Screen Shot 2022-04-14 at 13 57 44" src="https://user-images.githubusercontent.com/103438454/163361965-6faf9756-8eb7-4fae-adee-c7b91389c8f8.png">